cuegui/cuegui/MenuActions.py

```diff
@@ -1138,7 +1138,7 @@ def editSize(self, rpcObjects=None):
                    "contact the resource department."
             minSize = 0
             decimalPlaces = 0
-            (value, choice) = QtWidgets.QInputDialog.getDouble(self._caller, title, body, current,
+            (value, choice) = QtWidgets.QInputDialog.getDouble(self._caller, title, body, current/100.0,
```

Comment:
Remember that the API backend is still expecting values in hundredths of a core, so when you call `cuebotCall` below you need to multiple `value` by 100 there.

Also -- please update the [tests for editSize and editBurst](https://github.com/AcademySoftwareFoundation/OpenCue/blob/cbb98f444aa9c8df6e5246b6acb90734834845c5/cuegui/tests/MenuActions_tests.py#L1100) to assert that the correct value (i.e. the input * 100) is passed to setSize/setBurst.

To run the unit tests locally you can change to the `cuegui` directory then run `python setup.py test` .